src/site/markdown/JDK_MANAGEMENT.md

```diff
@@ -0,0 +1,597 @@
+<!---
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# JDK Management in Maven Wrapper
+
+This document describes the enhanced JDK management capabilities added to Maven Wrapper for the `only-script` distribution type, allowing projects to automatically download and manage JDK installations.
+
+## Overview
+
+The Maven Wrapper `only-script` distribution now supports:
+
+- 🚀 **Automatic JDK download and installation**
+- 📦 **JDK version management via maven-wrapper.properties**
+- 🔧 **Toolchain JDK support for multi-JDK builds**
+- 🔒 **SHA-256 checksum verification for security**
+- 🌍 **Cross-platform support (Windows, macOS, Linux)**
+- ⚙️ **Environment variable configuration**
+- ↩️ **Backward compatibility with existing configurations**
+
+## Distribution Type Support
+
+**Important**: JDK management is **only available for the `only-script` distribution type**. This design choice avoids the chicken-and-egg problem where Java is needed to download Java.
+
+```bash
+# Generate wrapper with JDK management support
+mvn wrapper:wrapper -Dtype=only-script -Djdk=17 -DjdkDistribution=temurin
+```
+
+The `only-script` distribution uses shell scripts (Unix) and PowerShell (Windows) to handle JDK download and installation directly, without requiring Java to be pre-installed.
+
+## Configuration
+
+### Basic JDK Configuration
+
+Add JDK configuration to your `.mvn/wrapper/maven-wrapper.properties` file:
+
+```properties
+# JDK Management
+jdkVersion=17                # Resolves to latest 17.x (e.g., 17.0.14)
+jdkDistribution=temurin      # Disco API distribution name
```

Comment:
Disco API was never mentioned before, not suddenly pops up in a comment. Should be mentioned before so people can look up whats available